### DISCUSSION TITLE
The `transient_prompt` option doesn't work in vsCode powershell

### DISCUSSION THREAD

**Discussion by babakfp** [JanDeDobbeleer/oh-my-posh#4863] [Troubleshoot] [answered]
Hi 👋

When I open Powershell (v7), the `transient_prompt` option works. But when I use Powershell in vsCode as a terminal, it doesn't work.

**Comment by JanDeDobbeleer** [comment, marked as answer]
Do you perhaps have shell integrations enabled?

**Reply by babakfp** [reply]
I added an option related to shell integration to the vsCode config and it fixed it. Later I removed the config and it continues to work! It works so 😄, thanks 🌸.
